src/Template/Element/index/table_actions.ctp

```diff
@@ -3,6 +3,19 @@ use Cake\Utility\Inflector;
 
 foreach ($actions['entity'] as $action) {
 
+    if ($action === 'add') {
+        continue;
+    }
+
+    if ($action === 'delete') {
```

Comment:
> From the PR

Hardcoding this seems like a bad idea. Maybe we should add something to the action class that signifies that requests need to happen with postLink?

Same for `add` action. I mentioned that to @jadb on IRC.